go.mod

```diff
@@ -1,6 +1,9 @@
 module github.com/terraform-providers/terraform-provider-archive
 
 require (
+	github.com/bmatcuk/doublestar v1.2.2
 	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/hashicorp/terraform-plugin-sdk v1.0.0
 )
+
+go 1.13
```

Comment:
This seems like it could be a problem to me, but I'm not sure if I can just remove this?

Reply:
Travis looks like it is building with go1.11.13.

You could try using the docker container for that version to build the go mod dependencies.

Reply:
I’ll give it a go! Thanks!

Reply:
So it looks like `github.com/bmatcuk/doublestar` requires go version 1.12 or later. Can we upgrade this project to use a more up-to-date version of go?

Reply:
I think it has been updated since this review. Can we try to get this merged now?